src/modules/translation/languages/en.json

```diff
@@ -99,5 +100,14 @@
         "length": "Description length must be between {min} and {max} characters."
       }
     }
+  },
+  "custom_layout": {
+    "title": "Custom",
+    "subtitle_one": "You can create custom layouts for your scene.",
+    "subtitle_two": "Enter values in the boxes below for the desired size.",
+    "max_area_error": "Your scene can't have more than {area} parcels",
```

Comment:
We will need to reuse these errors in the edit project modal. Let's move them to a more general domain,